runtime-sdk/src/dispatcher.rs

```diff
@@ -61,15 +63,29 @@ pub struct Dispatcher<R: Runtime> {
     abort_batch: Option<Arc<AtomicBool>>,
     /// Method registry.
     methods: MethodRegistry,
+    /// Handlers registered for consensus messages.
+    consensus_message_handlers: MessageHandlerRegistry,
 
     _runtime: PhantomData<R>,
 }
 
+const MODULE: &str = "dispatcher";
+
+/// State schema constants.
+pub mod state {
+    /// Map of message idx to message handlers for messages emitted in previous round.
+    pub const HANDLERS: &[u8] = &[0x01];
+}
```

Comment:
We should probably move this under the `core` module as that one currently has a similar thing for module migration metadata.